### ISSUE TITLE
Domain switcher slashcommand same as application switcher

### ISSUE THREAD

**Issue by VanWeapon** [arnoudkooi/ServiceNow-Utils#267]
the format for the payload is this, pretty similar to the app scope and update set switchers, 
```javascript
var payload = { "value": "global" }; //domain sys_id goes here
    fetch("/api/now/ui/concoursepicker/domain", {
        method: 'PUT',
        headers: {
          'Accept' : 'application/json, text/plain, */*',
          'Content-Type' : 'application/json;charset=UTF-8',
          'X-WantSessionNotificationMessages': false,
          'X-UserToken' : g_ck
        },
        body: JSON.stringify(payload),
      })
      .then(response => response.json())
.then(data => console.log(data));
```

would be cool to see a domain switcher slashcommand alongside application picker. 

I tried to shove the above script into a custom slashcommand override but I guess it's too big or something because it bugs out 

![image](https://user-images.githubusercontent.com/30616047/179188519-c3bc7d72-fdb0-4343-b2dd-f75b028aa1e9.png)
![image](https://user-images.githubusercontent.com/30616047/179188528-df1d3b1d-def0-49d7-9d34-0efa591b1bcb.png)

**Comment by arnoudkooi** [comment]
Did some testing, but looks like the domain list doesn't contain global, so that makes it more complicated as well...

**Comment by VanWeapon** [comment]
Yeah I noticed that as well, though the sys_id for the global domain is just "global" so it could be hardcoded I guess. Or I could make a separate slashcommand just to go straight to global

**Comment by VanWeapon** [comment]
Because realistically the use case is 9/10 times going to be to get out of my default domain and go to global

**Comment by arnoudkooi** [comment]
`/sd` should come available in 6.0.8.0
In case of domain it always adds global as option 0

When in a scope you cant query all domains, but I hope this covers your usecase.
<img width="350" alt="image" src="https://user-images.githubusercontent.com/840781/179845428-1c783b5f-1f31-484b-92e4-4b889d8c289b.png">

While at it also `/su` for updatesets is implemented. The switching should also be more reliable after this update.

**Comment by arnoudkooi** [comment]
6.0.8.0 now published https://twitter.com/sn_utils/status/1549714912817168385
